profiles/models.py

```diff
@@ -123,6 +130,13 @@ class Profile(models.Model):
     date_joined_micromasters = models.DateTimeField(blank=True, null=True, auto_now_add=True)
     linkedin = JSONField(blank=True, null=True)
     student_id = models.IntegerField(blank=True, null=True, unique=True)
+    currently_employed = models.CharField(
```

Comment:
these are temporary anyway, and just in this pr so that I have something to fiddle with in the frontend